addon/components/roadsigns-table.hbs

```diff
@@ -1,10 +1,10 @@
 <AuDataTable @isLoading={{@isLoading}} @noDataMessage="No data" @class="container-flex--contain" as |table|>
   <table.content as |c dataTable|>
     <c.header>
-      <AuDataTableThSortable @field="code" @label="Code" />
-      <AuDataTableThSortable @field="image" @label="Afbeelding" />
-      <AuDataTableThSortable @field="description" @label="Betekenis" />
-      <AuDataTableThSortable @field="category" @label="Category" />
+      <th class="u-c-data-table__header-title">Code</th>
```

Comment:
is there a reason for dropping the sort here?

Reply:
Because this component worked together with the data table and we are not using it to fetch the data, implementing the sortable header would have meant more work that I thought it wasn't worth it as the only really helpful sorting would be to sort by the code and we can already sort by that by default